### PR TITLE
Install Config update

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description
Expand install config info, and combine with EE and OSS.
- https://jira.mesosphere.com/browse/DOCS-1934

**New page:** https://github.com/dcos/dcos-docs/blob/1f7836faf17cfa727a36b34642d8ca7b662e10c1/1.9/installing/custom/configuration/index.md
image
**Updated page:**
https://github.com/dcos/dcos-docs/blob/1f7836faf17cfa727a36b34642d8ca7b662e10c1/1.9/installing/custom/configuration/configuration-parameters.md
image
**New page created from existing content:**
https://github.com/dcos/dcos-docs/blob/1f7836faf17cfa727a36b34642d8ca7b662e10c1/1.9/installing/custom/configuration/examples.md
image

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by lingmann** [comment]
I did a quick scan and this looks like a nice improvement. 👍

**Review by lingmann — APPROVED** [review]

**Comment by surdy** [comment]
This is good. Looks much more organized now. A good change :D

**Review by surdy — APPROVED** [review]

File: 1.9/installing/custom/configuration/configuration-parameters.md [hunk 1]
```diff
@@ -56,12 +136,88 @@ This parameter specifies the name of your cluster.
...
+            vtep_subnet: 44.128.0.0/20
+            vtep_mac_oui: 70:B3:D5:00:00:00
+            overlays:
+              - name: dcos
+                subnet: 9.0.0.0/8
+                prefix: 26
+        ```
+
+        *  `vtep_subnet` This parameter specifies a dedicated address space that is used for the VxLAN backend for the virtual network. This address space should not be routeable from outside the agents or master.
+        *  `vtep_mac_oui` This parameter specifies the MAC address of the interface connecting to it in the public node.
```
Comment (sascala) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: suggest: replace "it" with "the virtual network"

File: 1.9/installing/custom/configuration/configuration-parameters.md [hunk 2]
```diff
@@ -56,12 +136,88 @@ This parameter specifies the name of your cluster.
...
+            
+            **Important:** The last 3 bytes must be `00`.
+        *  __overlays__
+            *  `name` This parameter specifies the canonical name (see [limitations](/docs/1.9/networking/virtual-networks/) for constraints on naming virtual networks).
+            *  `subnet` This parameter specifies the subnet that is allocated to the virtual network.
+            *  `prefix` This parameter specifies the size of the subnet that is allocated to each agent and thus defines the number of agents on which the overlay can run. The size of the subnet is carved from the overlay subnet.
+
+ For more information see the [example](#overlay) and [documentation](/docs/1.9/networking/virtual-networks/).
```
Comment (sascala) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: comma after "information"

File: 1.9/installing/custom/configuration/configuration-parameters.md [hunk 3]
```diff
@@ -56,12 +136,88 @@ This parameter specifies the name of your cluster.
...
+ 
+### <a name="dns-search"></a>dns_search
+This parameter specifies a space-separated list of domains that are tried when an unqualified domain is entered (e.g. domain searches that do not contain &#8216;.&#8217;). The Linux implementation of `/etc/resolv.conf` restricts the maximum number of domains to 6 and the maximum number of characters the setting can have to 256. For more information, see <a href="http://man7.org/linux/man-pages/man5/resolv.conf.5.html">man /etc/resolv.conf</a>.
```
Comment (sascala) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: comma after "e.g."

File: 1.9/installing/custom/configuration/configuration-parameters.md [hunk 4]
```diff
@@ -56,12 +136,88 @@ This parameter specifies the name of your cluster.
...
+
+A `search` line with the specified contents is added to the `/etc/resolv.conf` file of every cluster host. `search` can do the same things as `domain` and is more extensible because multiple domains can be specified.
+
+In this example, `example.com` has public website `www.example.com` and all of the hosts in the datacenter have fully qualified domain names that end with `dc1.example.com`. One of the hosts in your datacenter has the hostname `foo.dc1.example.com`. If `dns_search` is set to &#8216;dc1.example.com example.com&#8217;, then every DC/OS host which does a name lookup of foo will get the A record for `foo.dc1.example.com`. If a machine looks up `www`, first `www.dc1.example.com` would be checked, but it does not exist, so the search would try the next domain, lookup `www.example.com`, find an A record, and then return it.
```
Comment (sascala) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: put "foo" in code markup?

change "lookup" in the last sentence to "look up".

File: 1.9/installing/custom/configuration/configuration-parameters.md [hunk 5]
```diff
@@ -121,64 +282,32 @@ This option specifies that Mesos agents are used to discover the masters by givi
...
 
-A `search` line with the specified contents is added to the `/etc/resolv.conf` file of every cluster host. `search` can do the same things as `domain` and is more extensible because multiple domains can be specified.
+### <a name="process_timeout"></a>process_timeout
+This parameter specifies the allowable amount of time, in seconds, for an action to begin after the process forks. This parameter is not the complete process time. The default value is 120 seconds.
 
-In this example, `example.com` has public website `www.example.com` and all of the hosts in the datacenter have fully qualified domain names that end with `dc1.example.com`. One of the hosts in your datacenter has the hostname `foo.dc1.example.com`. If `dns_search` is set to &#8216;dc1.example.com example.com&#8217;, then every DC/OS host which does a name lookup of foo will get the A record for `foo.dc1.example.com`. If a machine looks up `www`, first `www.dc1.example.com` would be checked, but it does not exist, so the search would try the next domain, lookup `www.example.com`, find an A record, and then return it.
+**Tip:** If have a slower network environment, consider changing to `process_timeout: 600`.
```
Comment (sascala) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: If _you_ have a slower network....

File: 1.9/installing/custom/configuration/configuration-parameters.md [hunk 6]
```diff
@@ -193,58 +322,46 @@ This required parameter specifies a YAML nested list (`-`) of DNS resolvers for
...
+- `security: strict`
 
-### <a name="check-time"></a>check_time
-This parameter specifies whether to check if Network Time Protocol (NTP) is enabled during DC/OS startup. It recommended that NTP is enabled for a production environment.
+Refer to the [Security modes](/1.9/security/#security-modes) section for a detailed discussion of each parameter. 
 
-*  `check_time: 'true'` Check if NTP is enabled during startup. You will receive an error if this is not enabled. This is the default value.
-*  `check_time: 'false'` Do not check if NTP is enabled during startup.
+### ssh_key_path
+This parameter specifies the path to the installer uses to log into the target nodes. By default this is set to `/genconf/ssh_key`. This parameter should not be changed because `/genconf` is local to the container that is running the installer, and is a mounted volume.
```
Comment (sascala) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: "This parameter specifies the path the installer uses..."

**Review by sascala — COMMENTED** [review]
general comment: you might consider removing "this parameter specifies" globally -- it's repeated a lot and I think doesn't add more information.

File: 1.9/installing/custom/configuration/configuration-parameters.md (hunk 3 above)
Comment (joel-hamill) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: i also added this to our style guide

File: 1.9/installing/custom/configuration/configuration-parameters.md (hunk 4 above)
Comment (joel-hamill) [inline 1.9/installing/custom/configuration/configuration-parameters.md]: @sascala seems like the jury is slightly in favor of using `lookup` as one word: 
- https://english.stackexchange.com/questions/290278/look-up-tables-look-up-tables-or-lookup-tables

**Comment by joel-hamill** [comment]
@sascala implemented your feedback. thanks!

**Pull request merged**